app/views/service/_svcs_show.html.haml

```diff
@@ -38,8 +40,10 @@
             .form-horizontal.static
               .form-group
                 .col-md-9
-                  - htm = job.raw_stdout('html').gsub('"', '\"').gsub("'", "\\\\'")
-                  %div{'ng-bind-html'=>"vm.sanitizeRenderHtml('#{htm}')"}
+                  - htm = @job.raw_stdout('html')
+                  %div#std_output2
+                    %miq-sanitize{:value => htm}
 :javascript
   miq_tabs_init('#services_tab');
-  miq_bootstrap('#service_details', 'sanitizeRender');
+  miq_bootstrap('#std_output', 'miq.helpers');
```

Comment:
@h-kataria - Just for my understanding as to what this code is doing:  is `miq_bootstrap` applying `miq.helpers` to the div id?